### PR TITLE
fix: Cargo bump version preserves `workspace_members`  in manifest

### PR DESCRIPTION
**Pull request by callumPearce**
* Calls to cargo_bump_version would remove the workspace_members entry in Cargo.toml for workspaces, this change ensures that workspace_members are instead preserved.

### CODE REVIEW

**Comment by callumPearce** [comment]
> Looks like we could've caught that with a unit test: Read, write and read again? Could you add one? 😄

Yeah great point, I'll look to add one now

File: src/tests/test_cargo_manifest.py [hunk 1]
```diff
@@ -34,6 +34,10 @@ def test_cargo_manifest_handles_unknown_fields_correctly() -> None:
 
 def test_cargo_manifest_writes_json_correctly() -> None:
     input_json = {
+        "members": [
```
Comment (callumPearce) [inline src/tests/test_cargo_manifest.py:37]: @NiklasRosenstein this ensures the `to_json` retains the workspace members

Comment (NiklasRosenstein) [reply src/tests/test_cargo_manifest.py]: Perfect

File: .changelog/_unreleased.toml [hunk 2]
```diff
@@ -1,3 +1,15 @@
+[[entries]]
+id = "3b3a2872-0721-4aad-bb66-85a92412f201"
+type = "feature"
+description = "Add `cargo_fmt(all_packages)` parameter"
```
[the remark this answers is not in the document]
Comment (callumPearce) [reply .changelog/_unreleased.toml:4]: Sorry, what's the issue here 😄 ?

[the remark this answers is not in the document]
Comment (NiklasRosenstein) [reply .changelog/_unreleased.toml:4]: I don't think this belongs into the PR 😄 This changelog entry was already added in https://github.com/kraken-build/kraken-std/pull/41/files

[the remark this answers is not in the document]
Comment (callumPearce) [reply .changelog/_unreleased.toml:4]: Ohhh right, my bad haha, removing it now

**Review by NiklasRosenstein — APPROVED** [review]

**Comment by tetigi** [comment]
🚂

**Pull request merged**